pkg/controller/controller_suite_test.go

```diff
@@ -0,0 +1,12 @@
+package controller_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"testing"
+)
```

Comment:
Why are we adding an empty test suite?
`peg` binary when run with a config file, acts as a test suite. We will use that to test it. Probably here: https://github.com/spectrocloud/peg/tree/add-libvirt

Better remove the suite from here.

Reply:
That just came in from the merge.